data/animals.js

```diff
@@ -1,14 +1,26 @@
 module.exports = [
   {
-    name: 'bessie',
-    cool_factor: 3,
+    id: 1,
```

Comment:
You won't need to include the id here since sql creates it automatically from your create tables file. This allows a user to add new items to the database without worrying about overlapping the ids (when we get to post in lab 7).